### PR TITLE
Mods

### PR DESCRIPTION
**Pull request by rhu1**
Minimal (temporary) revision to add menu links for Scribble-Java and Scribble-Scala.

Commented out old "User Documentation" (i.e., Web tooling) menu item and page.

### CODE REVIEW

File: src/main/jbake/templates/navigation.ftl [hunk 1]
```diff
@@ -20,20 +20,41 @@
               <a href="<#if (content.rootpath)??>${content.rootpath}<#else></#if>docs/overview/getting-started.html">Getting
                   Started</a>
             </li>
+                <li class="dropdown">
```
Comment (objectiser) [inline src/main/jbake/templates/navigation.ftl:23]: Minor - could be sorted later, but the indentation of this new block needs to be sorted.

**Review by objectiser — APPROVED** [review]
Minor comment about the indentation.

Main issue is the image referenced from the scribble-java.html - it seems to be user/Scribble-Java-Tutorials_files folder, but is not being referenced there I believe.

File: src/main/jbake/templates/navigation.ftl (hunk 1 above)
Comment (rhu1) [reply src/main/jbake/templates/navigation.ftl:23]: Have pushed fixes for broken file refs and the indentation.

**Comment by objectiser** [comment]
LGTM :+1:

**Pull request merged**